### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by BrianTKM**
Fixes 9 digit limit
Fixes deleting card bill using /delete /bankexpenditure
Removed isEmpty() and check for null

### CODE REVIEW

**Comment by BrianTKM** [comment]
Resolves #186

File: src/main/java/owlmoney/logic/command/transaction/DeleteDepositCommand.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public DeleteDepositCommand(String bankName, int index) {
      * @throws TransactionException If transaction is not a deposit.
      */
     public boolean execute(Profile profile, Ui ui) throws BankException, TransactionException {
-        profile.profileDeleteDeposit(this.expNumber, this.from, ui);
+        profile.profileDeleteDeposit(this.expNumber, this.from, ui, false);
```
Comment (shamussy) [inline src/main/java/owlmoney/logic/command/transaction/DeleteDepositCommand.java]: do you want to use expenditureNumber instead?

File: src/main/java/owlmoney/logic/parser/ParseRawData.java [hunk 2]
```diff
@@ -72,7 +72,7 @@ private int findEndIndex(int startIndex, String[] splitTypeSpecificArguments, St
     private String concatenateParameter(int startIndex, int endIndex, String[] splitTypeSpecificArguments) {
         StringBuilder individualParameter = new StringBuilder();
         for (int i = startIndex + NEXT_INDEX; i < endIndex; i++) {
-            if (individualParameter.toString().isEmpty() || individualParameter.toString().isBlank()) {
+            if (individualParameter.toString().isBlank()) {
```
Comment (shamussy) [inline src/main/java/owlmoney/logic/parser/ParseRawData.java]: should we check for null here?

File: src/main/java/owlmoney/logic/command/transaction/DeleteExpenditureCommand.java [hunk 3]
```diff
@@ -39,7 +39,7 @@ public DeleteExpenditureCommand(int index, String accountName, String type) {
      * @throws CardException        If card does not exist.
      */
     public boolean execute(Profile profile, Ui ui) throws BankException, TransactionException, CardException {
-        profile.profileDeleteExpenditure(this.expNumber, this.from, ui, this.type);
+        profile.profileDeleteExpenditure(this.expNumber, this.from, ui, this.type, false);
```
Comment (shamussy) [inline src/main/java/owlmoney/logic/command/transaction/DeleteExpenditureCommand.java]: do you want to use expenditureNumber instead?

**Pull request merged**